src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts

```diff
@@ -62,6 +62,8 @@ interface InstructorEditPanelDetail {
 })
 export class InstructorCourseEditPageComponent implements OnInit {
 
+  // @ts-ignore
+  @ViewChild('courseForm') form: any;
```

Comment:
I'm not sure if it is good practice to suppress ts errors. A potential solution is at https://stackoverflow.com/questions/56704164/angular-viewchild-error-expected-2-arguments-but-got-1. I also think that the use of `any` in typescript should be minimised.